README.md

```diff
@@ -0,0 +1,23 @@
+# stratisd
+
+A daemon that manages a pool of block devices to create flexible filesystems.
+
+## Background
+
+There are many benefits to volume-managing filesystems (VMFs) like ZFS and Btrfs. In contrast to traditional Unix filesystems, VMFs can span multiple block devices, and support multiple independent filesystem trees. (ZFS calls these datasets, Btrfs calls these subvolumes.)  VMFs can share space between trees using copy-on-write, and support using their multiple block devices to provide RAID-style protection from data loss.
+
+Stratis (which includes [stratisd](https://github.com/stratis-storage/stratisd) as well as [stratis-cli](https://github.com/stratis-storage/stratis-cli)), attempts to provide VMF-style features, but by integrating layers of existing technology, such as Linux's devicemapper subsystem, and a non-VMF high-performance filesystem like XFS. `stratisd` manages collections of block devices, and exports a D-Bus API. Stratis-cli's `stratis` provides a conventional command-line tool for the user, instead of D-Bus directly.
```

Comment:
Change ", but by" to "by", 'cause that's just how we do it (no buts).

Change "Stratis-cli's `stratis` provides a conventional command-line tool for the user, instead of D-Bus directly." to
"Stratis-cli's `stratis` provides a command-line tool which itself uses the D-Bus API  to communicate with `stratisd`.